src/main/java/labs/introtoprogramming/lab5/object/Parser/StringUtils.java

```diff
@@ -0,0 +1,89 @@
+package labs.introtoprogramming.lab5.object.Parser;
+
+import java.util.logging.Logger;
+
+import static java.util.logging.Level.SEVERE;
+
+public class StringUtils {
+    private static Logger log = Logger.getLogger(StringUtils.class.getName());
+
+    public static void printErrMsg(String methodName, String errorMsg, int mCount, char message[]) {
```

Comment:
This function is only used in skipWhiteSpace

Reply:
done